README.md

```diff
@@ -33,17 +33,27 @@ The development is at very early stage. There are many TODO items:
 - [ ] Use async/await to avoid blocking operations
 - [ ] More CLI subcommands
   - [x] `create`
+  - [x] `dump`
   - [x] `list`
   - [ ] `extract`
 
 ## Command line tool
 
 This repository also contains a command line tool, called `webbundle`.
-
 To install `webbundle` command, run the following:
 
 ```shell
 cargo install --features=cli webbundle
 ```
 
-See `webbundle --help` for the usage.
+### create
+```no_run
```

Comment:
Nit; We don't need `no_run` for code blocks in README.md because `no_run` is only meaningful in `cargo test`.
Would you use `shell` for consistency in this file?

Reply:
@hayatoito my mistake 😢 I'll fix during today.

Reply:
@hayatoito fixed~

Reply:
Thanks! 👍